sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1580,6 +1580,13 @@ private synchronized void showUpdateSetupFailedDialog() {
         if (!shouldRefreshDialog(mUpdateSetupFailedDialog)) {
             return;
         }
+
+        /* Check if foreground activity is null, if null we return, if not we proceed to show the setup failed dialog */
```

Comment:
I would avoid such comments, if the written code is clean and easily understandable :)

Reply:
done